python/ray/dashboard/memory_utils.py

```diff
@@ -97,9 +97,7 @@ def __init__(
         self.task_status = object_ref.get("taskStatus", "?")
         if self.task_status == "NIL":
             self.task_status = "-"
-        self.attempt_number = int(object_ref.get("attemptNumber", 0))
-        if self.attempt_number > 0:
-            self.task_status = f"Attempt #{self.attempt_number + 1}: {self.task_status}"
+        self.attempt_number = int(object_ref.get("attemptNumber", 0)) + 1
```

Comment:
hmm previously we have +1 to be human friendly. Now that we make it programmically readable I wonder if we can be 0-indexed.

Reply:
I keep it +1 to be compatible with the previously displayed information. Personally I think we should still keep the same to avoid confusion. But I don't have a strong opinion. 

I also added the description in the schemas to make clear that the attempt number includes the current execution.

Reply:
ok I think 1 index is ok. merging